dist/index.html

```diff
@@ -0,0 +1,165 @@
+<!--
+*  macOSNotifJS: A simple Javascript plugin to create simulated macOS notifications on your website.
+*  <https://github.com/MattIPv4/macOSNotifJS/>
+*  Copyright (C) 2018 Matt Cowley (MattIPv4) (me@mattcowley.co.uk)
+*
+*  This program is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU Affero General Public License as published
+*   by the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*  This program is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*  You should have received a copy of the GNU Affero General Public License
+*   along with this program. If not, please see
+*   <https://github.com/MattIPv4/macOSNotifJS/blob/master/LICENSE> or <http://www.gnu.org/licenses/>.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>macOSNotifJS</title>
+    <style>
+        html, body {
+            margin: 0;
+            background: #fff;
+            color: #000
+        }
+
+        html {
+            padding: 0;
+        }
+
+        body {
+            padding: 3em 5em;
+        }
+
+        h1 {
+            text-align: center;
+        }
+
+        h1 small {
+            font-weight: normal;
+            font-size: 50%;
+        }
+
+        code, pre {
+            color: rgb(255, 59, 48);
+        }
+
+        button {
+            background: rgb(0, 122, 255);
+            border: none;
+            border-radius: 10px;
+            font-weight: normal;
+            color: #fff;
+            font-size: 1.2em;
+            line-height: 1.5em;
+            margin-bottom: 0.5em;
+        }
+
+        button:hover {
+            background: rgb(0, 111, 230);
+        }
+
+        button:active {
+            background: rgb(0, 99, 204);
+        }
+
+        button + code {
+            margin-left: 0.5em;
+        }
+    </style>
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="macOSNotif.css" rel="stylesheet"><script type="text/javascript" src="macOSNotif.min.js"></script></head>
```

Comment:
Can this be tidied to have new lines and conform to XHTML with self-closing slashes?
Script should also be at the end of the body not in the head?